### PR TITLE
GCCcore: Add includes and type spellings for newer glibc

### PR DESCRIPTION
**Pull request by zao**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/g/GCCcore/GCCcore-5.3.0.eb [hunk 1]
```diff
@@ -15,7 +15,7 @@ source_urls = [
     'http://ftpmirror.gnu.org/gnu/gcc/gcc-%(version)s',  # GCC auto-resolving HTTP mirror
     'http://ftpmirror.gnu.org/gnu/gmp',  # idem for GMP
     'http://ftpmirror.gnu.org/gnu/mpfr',  # idem for MPFR
-    'http://ftpmirror.gnu.org/gnu/mpc',  # idem for MPC
+    'http://www.multiprecision.org/mpc/download',  # MPC official
```
Comment (boegel) [inline easybuild/easyconfigs/g/GCCcore/GCCcore-5.3.0.eb]: This URL is not correct anymore, we changed this in #5662.

Please sync your branch with current `develop`?

File: easybuild/easyconfigs/g/GCCcore/GCCcore-5.x-asan-fix-missing-include-signal-h.patch [hunk 2]
```diff
@@ -0,0 +1,12 @@
+diff --git a/libsanitizer/asan/asan_linux.cc b/libsanitizer/asan/asan_linux.cc
```
Comment (boegel) [inline easybuild/easyconfigs/g/GCCcore/GCCcore-5.x-asan-fix-missing-include-signal-h.patch]: @zao Please mention upstream source or include author + include short description of what this fixes (bonus points if you include the exact error message)

File: easybuild/easyconfigs/g/GCCcore/GCCcore-5.x-fix-gperf-signatures.patch [hunk 3]
```diff
@@ -0,0 +1,125 @@
+diff --git a/gcc/cp/Make-lang.in b/gcc/cp/Make-lang.in
```
Comment (boegel) [inline easybuild/easyconfigs/g/GCCcore/GCCcore-5.x-fix-gperf-signatures.patch]: @zao Missing: short description + upstream source or author of this patch

File: easybuild/easyconfigs/g/GCCcore/GCCcore-6.x-fix-glibc-types.patch [hunk 4]
```diff
@@ -0,0 +1,79 @@
+diff -ru gcc-6.4.0-orig/libgcc/config/i386/linux-unwind.h gcc-6.4.0/libgcc/config/i386/linux-unwind.h
```
Comment (boegel) [inline easybuild/easyconfigs/g/GCCcore/GCCcore-6.x-fix-glibc-types.patch]: @zao Missing: short description + upstream source or author of this patch

File: easybuild/easyconfigs/g/GCCcore/GCCcore-5.3.0.eb (hunk 1 above)
Comment (zao) [reply easybuild/easyconfigs/g/GCCcore/GCCcore-5.3.0.eb]: Do I need to run full `develop` EasyBuild for this, or does it suffice to manipulate the repo in `git-working-dirs-path`?

Comment (boegel) [reply easybuild/easyconfigs/g/GCCcore/GCCcore-5.3.0.eb]: You don't need to run full `develop` EasyBuild to do that, but you do need to use `git` commands currently (check out the current `develop` branch, `git merge develop` into the branch for your PR, `git push` the updated branch). We should probably look into `eb --sync-pr-develop` or something...

I'll send a PR your way to handle this for now.

Comment (zao) [reply easybuild/easyconfigs/g/GCCcore/GCCcore-5.3.0.eb]: Great, I'm in the process of addressing the feedback and figuring out where my patches came from.

Comment (boegel) [reply easybuild/easyconfigs/g/GCCcore/GCCcore-5.3.0.eb]: @zao issues with MPC source URLs fixed in https://github.com/zao/easybuild-easyconfigs/pull/1

That PR is going to be impossible to review, but if you want to review it, just look at https://github.com/zao/easybuild-easyconfigs/pull/1/commits/b83c5e849baa000c85832c385a31d05d12a01e75

**Comment by boegel** [comment]
postponing this for EB v3.6.0, as discussed with @zao...

**Comment by zao** [comment]
Been way too busy with other stuff to care about these, particularly as upstream lua-posix (and thus Lmod) is still broken on Bionic.
No idea about on what timeframe I can rebuild the EB tree on my test rig as it has been paved.

**Comment by migueldiascosta** [comment]
@zao ping?

**Comment by boegel** [comment]
No follow-up, and quite old `GCCcore` easyconfigs, so closing this as is...

**Pull request closed without merging**